Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'addressable'
 gem 'unicorn', '4.6.2'
 gem 'kaminari', '0.15.1'
 gem 'bootstrap-kaminari-views'
-gem 'gds-api-adapters', '18.2.0'
+gem 'gds-api-adapters', '18.2.0', git: 'https://github.com/alphagov/gds-api-adapters', branch: 'add_draft_item_test_assertion'
```

Comment:
I'll rebase this once https://github.com/alphagov/gds-api-adapters/pull/292 has been merged and a new version released.